### ISSUE TITLE
Add Option to Run QSpy in Non-Interactive Mode (Linux)

### ISSUE THREAD

**Issue by DanielLyubin** [QuantumLeaps/qtools#13]
As a part of my CI process I run QSpy in the background. It fails with error "<CONS> ERROR    getting terminal attributes".
I added the error message generated from the errno after executing the `tcgetattr` function and received the error "inappropriate ioctl for device". I ran QSpy on the same machine but from a user shell and it worked. Tried to redirect the input stream for QSpy to /dev/null and got the same message I got from the CI pipeline ("inappropriate ioctl for device"). 

Long story short - CI pipelines don't have an `stdin`, and QSpy always needs an input, so I need a new option from QSpy to run in non-interactive mode and not expect an input (add a "-ni" flag for example to state non-interactive mode).

Thank you.

**Comment by quantum-leaps** [comment]
A new command-line option (-k) has been added to QSPY host app. This option will suppress keyboard input and configuring the terminal that runs QSPY. 

Please note that without the keyboard (-k option active), you cannot terminate QSPY in the usual way of typing 'x' or 'Esc'. But you can still type Ctrl-C or, you can kill the process running QSPY.

QSPY now intercepts the Kill signal and will perform cleanup (closing any open files, etc.). But note that this must be simple kill. The unconditional kill (kill -9 ...) is not interceptable and therefore it is NOT recommended for terminating QSPY.

The changes are available in the repository (as commit "6.9.4rc2")

--MMS

**Comment by quantum-leaps** [comment]
This issue is closed as already implemented (QSPY 6.9.3).
--MMS
